Woof/Woof/Helper/AppStyle/FontSize.swift

```diff
@@ -0,0 +1,13 @@
+import SwiftUI
+
+extension AppStyle {
+    /// Provides the font sizes used within the application.
+    enum FontSize {
```

Comment:
this is not `size`, but `style` probably:
```suggestion
    enum FontStyle {
```

Reply:
Fixed with https://github.com/ios-course/ironfoudation-team-project/commit/f6ce719f1bf56b406708b8bce981f36ca98173c0